### PR TITLE
Bugfix/unitless gutter on masonry column

### PR DESCRIPTION
**Pull request by peterramsing**
**What kind of change is this? (Bug Fix, Feature...)**
🐞

**What is the current behavior (You can also link to an issue)**
#383

**What is the new behavior this introduces (if any)**

**Does this introduce any breaking changes?**
Shouldn't

**Does the PR fulfill these requirements?**
- [x] Tests for the changes have been added
- [N/A] Docs have been added or updated

### CODE REVIEW

**Comment by peterramsing** [comment]
I'm fixing up a bunch of the code in here while I'm here. Noticed there were a few dead lines and more test coverage needs. I have on more thing to check after this commit but I need to head home from the coffee shop. I think that `ln27` is a dead line but I'm not sure.

**Comment by peterramsing** [comment]
This should be good to go, now.

**Comment by peterramsing** [comment]
@codebysubtract If you want to glance at this and make sure I'm not crazy we can get this in. 👍

File: test/lost-masonry-column.js [hunk 1]
```diff
@@ -3,36 +3,70 @@
...
+      );
+    });
 
-  it('supports no gutter', function() {
-    check(
-      'a { lost-masonry-column: 0; }',
-      'a { float: left; width: calc(99.9% * 0 - 30px); margin-left: 15px;' +
-      ' margin-right: 15px; }'
-    );
+    it('supports a null unit for "no gutter"', function() {
```
Comment (steve-holland) [inline test/lost-masonry-column.js]: I'm not sure you need a separate test here. I think you could add the `check` done here to the test above `supports no gutter`. You are essentially checking the same thing, just whether they have added the unit to the end or not. Only very minor thing, so by no means essential.

**Review by steve-holland — CHANGES_REQUESTED** [review]
Looking good overall. Just a minor thing in the tests, which may come down to personal preference, so I won't mind if you disagree :)

**Comment by peterramsing** [comment]
@codebysubtract Thanks! Yeah, those could certainly be merged together. Done. Gonna merge. 👍

**Pull request merged**